api/http/handler/users/user_update.go

```diff
@@ -57,6 +57,10 @@ func (handler *Handler) userUpdate(w http.ResponseWriter, r *http.Request) *http
 		return &httperror.HandlerError{http.StatusBadRequest, "Invalid user identifier route variable", err}
 	}
 
+	if handler.isDemo && userID == 1 {
```

Comment:
probably check if user role is admin instead?

Reply:
the requirement was to disable changing the initial admin user

Reply:
added a field "Readonly" that will disable editing this user object